.changeset/lucky-rocks-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/preview-middleware': patch
+---
+
+Adds LocalStorageConnector to sandbox.html
```

Comment:
could we please change it "Adds local persistence of personalizations across local preview sessions" or something along the lines to better explain what the change does?

Reply:
Thanks, done with https://github.com/SAP/open-ux-tools/pull/1381/commits/53c6787393b5f67aee391b4bb895885ab3d15787